modules/ztp-clusterinstance-argocd-migrate.adoc

```diff
@@ -0,0 +1,177 @@
+// Module included in the following assemblies:
+//
+// * edge_computing/ztp-migrate-clusterinstance.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ztp-clusterinstance-argocd-migrate_{context}"]
+= Migrating the cluster to ClusterInstance CRs using ArgoCD
+
+After you have prepared your Git repository with the new `ClusterInstance` custom resource (CR) and any associated resources, you can configure ArgoCD on the hub cluster.
+
+When you apply the ArgoCD application, the SiteConfig Operator reconciles the new ClusterInstance CRs against the existing cluster configuration. As the configuration is the same between the `SiteConfig` CR and the `ClusterInstance` CR, the target cluster seemlessly migrates to using the `ClusterInstance` CR.
+
+// is this seemless? Any reboots or downtime?
+
+//confusion about what happens to other managed clusters when you delete the ArgoCD application, are all managed clusters then "paused" from syncing while the migration is happening? 
+
```

Comment:
@sudomakeinstall2 - Just checking no downtime is expected. And also a general question, probably due to my limited knowledge, but if ArgoCD is deleted, does that mean all managed clusters cannot be sync'd while the migration preparation is happening? Does 1 ArgoCD app control all clusters, or is it a 1-1 between ArgoCD and a cluster.